enumflags/src/lib.rs

```diff
@@ -361,3 +381,26 @@ mod impl_serde {
         }
     }
 }
+
+#[derive(Debug, Copy, Clone)]
+pub struct FromBitsError<T: RawBitFlags> {
+    flags: BitFlags<T>,
+    invalid: T::Type,
+}
```

Comment:
I think this belongs to the `fallible` module. It could be reexported at the top level.